stable_baselines3/common/vec_env/vec_transpose.py

```diff
@@ -46,6 +46,13 @@ def transpose_image(image: np.ndarray) -> np.ndarray:
 
     def step_wait(self) -> VecEnvStepReturn:
         observations, rewards, dones, infos = self.venv.step_wait()
+        terminal_key = "terminal_observation"
```

Comment:
Good catch. This seems to have been a bug in original version? If so, please mention in the docs that it fixes this.